verifier/helpers.go

```diff
@@ -0,0 +1,87 @@
+package verifier
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/notaryproject/notation-core-go/signature"
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation-go/verifier/trustpolicy"
+	"github.com/notaryproject/notation-go/verifier/truststore"
+)
+
+func loadPolicyDocument() (*trustpolicy.Document, error) {
+	policyDocument := &trustpolicy.Document{}
+	jsonFile, err := dir.ConfigFS().Open(dir.PathTrustPolicy)
+	if err != nil {
+		return nil, err
+	}
+	defer jsonFile.Close()
+	err = json.NewDecoder(jsonFile).Decode(policyDocument)
+	if err != nil {
+		return nil, err
+	}
+	return policyDocument, nil
+}
+
+func loadX509TrustStores(ctx context.Context, scheme signature.SigningScheme, policy *trustpolicy.TrustPolicy) ([]*x509.Certificate, error) {
```

Comment:
Do we still need this method when we have `GetCertificates` in truststore.go?

Reply:
loadX509TrustStores is an unexported helper function to parse the truststores inside the trust policy statement. GetCertificates takes store type and named store to get the certificates in each named store. So we still need loadX509TrustStores to parse the trust policy statement.